pandas/tests/arrays/test_timedeltas.py

```diff
@@ -290,8 +290,18 @@ def test_sum_2d_skipna_false(self):
         )._values
         tm.assert_timedelta_array_equal(result, expected)
 
-    def test_std(self):
-        tdi = pd.TimedeltaIndex(["0H", "4H", "NaT", "4H", "0H", "2H"])
+    # Adding a Timestamp makes this a test for DatetimeArray.std
```

Comment:
shouldn't this be in test_datetimes? yes i read your comment, but still the location is odd